components/src/LiveCode/index.tsx

```diff
@@ -0,0 +1,24 @@
+/** @jsx jsx */
```

Comment:
Remember to import by default our scope from the `liveCodeScope`, and if the user passes a scope, they should be merged.

```jsx
import globalScope from './liveCodeScope'`

...

export const LiveCode = ({ scope = {}, code }: Props) => (
  <LiveProvider scope={...globalScope, ...scope} code={code}>
  ...
)
```

Reply:
:rocket: